### PR TITLE
fix(test) : links and tree expansion tests

### PR DESCRIPTION
**Pull request by Raunak1203**
### [ ] What does this PR do? _[Mandatory]_
This PR adds workitem link and tree expansion test

### [ ] What issue/task does this PR references? _[Mandatory]_
https://github.com/fabric8-ui/fabric8-planner/issues/2562 and https://github.com/fabric8-ui/fabric8-planner/issues/2566

### [ ] Are the tests Included? _[Mandatory]_
Yes

### CODE REVIEW

**Comment by jarifibrahim** [comment]
[test]

**Comment by fabric8cd** [comment]
@Raunak1203 fabric8/fabric8-ui:SNAPSHOT-PR-2563-8 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2563-fabric8-planner.badger.fabric8.io

**Comment by jarifibrahim** [comment]
[test]

**Comment by fabric8cd** [comment]
@Raunak1203 fabric8/fabric8-ui:SNAPSHOT-PR-2563-9 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2563-fabric8-planner.badger.fabric8.io

File: src/tests/specs/smoke/smokeTest.spec.ts [hunk 1]
```diff
@@ -139,11 +139,11 @@ describe('Planner Smoke Tests:', () => {
     expect(await planner.quickPreview.hasComment('new comment')).toBeFalsy();
   });
 
-  xit('Create custom query', async() => {
+  it('Create custom query', async() => {
     await planner.sidePanel.clickRequirement();
     await planner.header.selectFilter('State','in progress');
     await planner.header.saveFilters('Query 1');
-    expect(await planner.sidePanel.hasCustomQuery('Query 1')).toBeTruthy();
+    expect(await planner.sidePanel.getMyFiltersList()).toContain('Query 1');
```
Comment (nimishamukherjee) [inline src/tests/specs/smoke/smokeTest.spec.ts]: hasCustomQuery('Query 1')).toBeTruthy();
and 
getMyFiltersList()).toContain('Query 1');
test for the same thing.
Why should we use getMyFiltersList()).toContain('Query 1') ?

Comment (jarifibrahim) [reply src/tests/specs/smoke/smokeTest.spec.ts]: @nimishamukherjee The problem with using `hasCustomQuery()` methods is that they're difficult to debug. When you use the `hasCustomQuery()` method and the tests fails, you see 
`Expected false to be true`. This output is useless if you wish to know why the tests failed.
Instead, when we use the `getMyFiltersList()` we have output like `Expected ['a', 'b', 'c'] to contain 'd'` and you know why the tests failed.

We will be refactoring all the `has...()` methods to `get...()` method.

**Comment by jarifibrahim** [comment]
[test]

**Comment by fabric8cd** [comment]
@Raunak1203 fabric8/fabric8-ui:SNAPSHOT-PR-2563-10 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2563-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@Raunak1203 fabric8/fabric8-ui:SNAPSHOT-PR-2563-11 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2563-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@Raunak1203 fabric8/fabric8-ui:SNAPSHOT-PR-2563-12 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2563-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@Raunak1203 fabric8/fabric8-ui:SNAPSHOT-PR-2563-13 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2563-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@Raunak1203 fabric8/fabric8-ui:SNAPSHOT-PR-2563-14 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2563-fabric8-planner.badger.fabric8.io

**Pull request merged**